21-MergeTwoSortedLists

```diff
@@ -0,0 +1,71 @@
+/*
+https://leetcode.com/problems/merge-two-sorted-lists/
+
+Merge two sorted linked lists and return it as a new list. The new list should be made by splicing together the nodes of the first two lists.
+
+Example:
+
+Input: 1->2->4, 1->3->4
+Output: 1->1->2->3->4->4
+*/
+
+class ListNode {
```

Comment:
Usually, these are already defined in the leet code source code, please comment it out here, so if some1 wants to copy and paste, It should work without any changes